Templates/settings.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends($layout)
+
+@section('content')
+
+<x-global::pageheader :icon="'fa fa-puzzle-piece'">
+    <h1>OmniSearch Settings</h1>
+</x-global::pageheader>
+
+
+<div class="maincontent">
+<?php echo $tpl->displayNotification(); ?>
+    <div class="maincontentinner">
+        <h5 class="subtitle">Omnisearch Settings</h5>
+        <p style="padding-bottom: 15px">These settings will change the way the OmniSearch plugin works.</p>
+        <h4 class="widgettitle title-light"><span class="fa fa-cog"></span>
+            Cache expiration durations
+        </h4>
+        <form class="" method="post" id="" action="<?=BASE_URL ?>/OmniSearch/settings">
```

Comment:
Maybe strip this file from omnisearchstuff

Reply:
Yeah. Thought to keep it as a reference for how to set settings up nicely, but makes less sense to keep now in git.